nodes/viz/texture_viewer.py

```diff
@@ -330,7 +330,13 @@ def draw_buttons_ext(self, context, layout):
         rightside.operator(directory_select, text="", icon='IMASEL').fn_name = "set_dir"
 
     def draw_label(self):
-        return (self.label or self.name) + ' ' + str(size_tex_dict.get(self.selected_mode)) + "^2"
+        if self.selected_custom_tex:
+            width = self.inputs['Width'].sv_get(deepcopy=False)[0][0]
```

Comment:
i will compact this with a `width = get_from_c_size(self, 'Width')`

```python
def get_from_c_size(self, string):
        if string is 'Width':
            data = self.inputs['Width'].sv_get(deepcopy=False)[0][0]
        else:
            data = self.inputs['Height'].sv_get(deepcopy=False)[0][0]
        return data
```

not tested yet.